dali/test/python/test_operator_readers_webdataset_base.py

```diff
@@ -0,0 +1,112 @@
+import nvidia.dali as dali
+from nvidia.dali import pipeline_def
+import nvidia.dali.fn as fn
+import nvidia.dali.fn.readers as readers
+from test_utils import compare_pipelines, get_dali_extra_path
+from nose.tools import assert_raises, assert_equal
```

Comment:
Please use `nose_utils` for `assert_raises` and check the returned error message as well.